code/stable/dblpend/SRS/PDF/DblPend_SRS.tex

```diff
@@ -67,13 +67,13 @@ \subsection{Table of Symbols}
 {colspec={l l l}, rowhead=1, hline{1,Z}=\heavyrulewidth, hline{2}=\lightrulewidth}
 \textbf{Symbol} & \textbf{Description} & \textbf{Units}
 \\
-$a_{\text{x}1}$ & Horizontal acceleration of the first object & $\frac{\text{m}}{\text{s}^{2}}$
+${a_{\text{x}1}}$ & Horizontal acceleration of the first object & $\frac{\text{m}}{\text{s}^{2}}$
```

Comment:
Same here. So I think the logic of when to add curlies still needs tweaking.

Reply:
I _think_ that this comment (and the one above) were made on a commit that reverts my earlier removal of those curly braces. The current diff doesn't show these changes happening, so it's what's currently on `main`.

I agree that this is bad however. I want the extra curly braces removed. This is where an actual design is needed (#1606).

Reply:
I _think_ that this comment (and the one above) were made on a commit that reverts my earlier removal of those curly braces. The current diff doesn't show these changes happening, so it's what's currently on `main`.

I agree that this is bad however. I want the extra curly braces removed. This is where an actual design is needed (#1606).